### ISSUE TITLE
Somtimes predicted categories don't sum to one (overlap + ensembled)

### ISSUE THREAD

**Issue by alisandra** [weberlab-hhu/Helixer#4]
For instance papio_anubis, chunk 479

![](https://user-images.githubusercontent.com/11819588/81834525-546c0680-9541-11ea-827c-d23b811db236.png)

@soi explained this well in the original (GenuFF repo) issue:
"I believe this is due to the way overlapping is done. Could this chunk be the last one in a chromosome? I vaguely remember that such things happened, but that it also had no effect on the argmax outcome (hopefully).

Basically there is nothing to overlap with at the end and the initial array to which the sequences get copied to for the final averaging is initialized with zeros, so the average shrinks but not the argmax."
